### PR TITLE
[Blocked] Feat: integration with pulse and getting merkle proof from mAPI

### PR DESCRIPTION
**Pull request by vitalibalashka**
Pulse will fire a webhook whenever a new blockheader arrives. And transactions without merkle proof will be updated with data from mAPI

### CODE REVIEW

**Comment by vitalibalashka** [comment]
@mrz1836 there is a question: centrifuge 0.9.6 is required for pulse integration. And now we have 2 versions of centrifuge in bux. And pulse's centrifuge added via
`replace github.com/centrifugal/centrifuge-go/0.9.6 => github.com/centrifugal/centrifuge-go v0.9.6`
So from now every project which will use bux also need this mentioned replace. Perhaps it would be better to create a fork for this particular version of the centrifuge, so as not to add this replace everywhere?

File: model_transactions.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ type Transaction struct {
 	TotalValue      uint64          `json:"total_value" toml:"total_value" yaml:"total_value" gorm:"<-create;type:bigint" bson:"total_value,omitempty"`
 	XpubMetadata    XpubMetadata    `json:"-" toml:"xpub_metadata" gorm:"<-;type:json;xpub_id specific metadata" bson:"xpub_metadata,omitempty"`
 	XpubOutputValue XpubOutputValue `json:"-" toml:"xpub_output_value" gorm:"<-;type:json;xpub_id specific value" bson:"xpub_output_value,omitempty"`
+	MerkleProof     MerkleProof     `json:"merkle_proof" toml:"merkle_proof" yaml:"merkle_proof" gorm:"<-create;type:text;comment:Merkle Proof payload from mAPI" bson:"merkle_proof,omitempty"`
```
Comment (mrz1836) [inline model_transactions.go:59]: I think this Gorm tag is wrong. It says it will only set the value on "insert/create"

**Review by mrz1836 — CHANGES_REQUESTED** [review]
See my comments - thanks!

File: model_transactions.go (hunk 1 above)
Comment (vitalibalashka) [reply model_transactions.go]: Fixed

**Comment by vitalibalashka** [comment]
@mrz1836 I think we could make a refactoring in woc library. But first we need to wait for the moment when the Taal will fix their monitor agent

**Pull request closed without merging**